app/Models/Log.php

```diff
@@ -36,13 +39,14 @@
 class Log extends Model
```

Comment:
Using the Log way seems pretty good. Although we might want to split up verbose logging from other logging. An admin isn't interested in wheter an user was succesful in editing their own settings. But is however interested in who edited the FAQ, who signed someone in, who changed the FAQs.

But this verbose logging might be helpful for us as developers. If we go this route, we need to think of a way to better split up these logs.

Reply:
Instantiating these logs also seems quite error prone, maybe we can like use a static function in Log to create it for us? Then it might be easier for us to enforce certain parameters, and make sure you can't make mistakes.